gatsby-config.js

```diff
@@ -91,19 +91,19 @@ module.exports = {
         preferCanvas: true,
         maxWidth: '100%',
         maxHeight: 600,
-      },
+      }
     },
     {
       resolve: 'gatsby-source-graphql',
       options: {
-        typeName: process.env.GATSBY_HASURA_GRAPHQL_TYPE_NAME,
-        fieldName: process.env.GATSBY_HASURA_GRAPHQL_FIELD_NAME,
-        url: process.env.GATSBY_HASURA_GRAPHQL_URL,
-        refetchInterval: 60,
+        typeName: 'MyQuery',
```

Comment:
astea de ce le-ai revert? erau ok in env files

Reply:
Eroare in consola la gatsby develop
```GraphQL request:3:9
2 |       query {
3 |         hasura {
  |         ^
4 |           providers {
```